### PR TITLE
unify syntax in readmes

### PR DESCRIPTION
**Pull request by petrparolek**

### CODE REVIEW

**Comment by paveljanda** [comment]
Is there any indentation  convention when using neon files?

I personally use tabs. Therefor I would prefer using tabs also in here.

File: .docs/README.md [hunk 1]
```diff
@@ -19,39 +19,39 @@ composer require contributte/jsonrpc
...
+      </td>
+      <td align="center">
+        <a href="https://github.com/gameeapp">
+            <img width="150" height="150" src="https://avatars3.githubusercontent.com/u/13903740?s=200&v=4">
+        </a>
+        </br>
+        <a href="https://github.com/gameeapp">Gameeapp</a>
+      </td>
+    </tr>
+  </tbody>
```
Comment (paveljanda) [inline .docs/README.md:54]: Would it be pls possible to keep the tabs? I do not use spaces in html at all.. thx!

Comment (petrparolek) [reply .docs/README.md:54]: Promin, nebudu pořád dokola hodiny synchronizovat přes 130 repositářů!

Comment (f3l1x) [reply .docs/README.md:54]: Just keep tabs @petrparolek.

Comment (paveljanda) [reply .docs/README.md:54]: You know what - I am probably older so I should step back.. I didn't notice it is only the maintainers block. 👍

**Pull request merged**